gflib/text.h

```diff
@@ -7,6 +7,41 @@
 // loaded at once but not copied to vram yet.
 #define TEXT_SKIP_DRAW 0xFF
 
+/*
+Enable automatic decapitalization of *all* text
+Exceptions:
+- Several bigrams: TV, TM, HP, HM, PC, PP, PM
+- Player names, nicknames, box names
+- Strings beginning with {FIXED_CASE}:
+  - C strings that use `_C` or `__C`
+  - ASM strings that use `.fixstr`
+- If mirroring enabled, string addresses passed through MirrorPtr
+*/
+#define DECAP_ENABLED TRUE
```

Comment:
I think it'd be better to move the configs to a file in the `include/config/` folder for easier accesibility.

Reply:
Are those files included where they need to be? I wasn't sure if it would work if the config file isn't included in text.h or the place where I add a struct field

Reply:
They're added in `include/constant/global.h`.
Also the battle and item ones are included in some of the asm script files, meaning that only constants should be added  for them to be used in scripts.

Reply:
Does that still work for header files like text.h, where some macros have to be defined based on DECAP_MIRRORING, etc?

Reply:
As long as both `include/global.h` and `include/text.h` are included in the file where they're used (which is very unlikely that they're not), preproc should resolve it.